spec/features/publish/courses/editing_course_funding_type_spec.rb

```diff
@@ -24,25 +24,24 @@
     end
 
     context 'copying content from another course' do
-      let!(:course2) do
+      let!(:biology_course) do
```

Comment:
I had a rubocop failure about using indexes in the let statements (2, 3), so that's why this is changed.